src/dag/key.rs

```diff
@@ -92,10 +90,10 @@ mod tests {
     #[test]
     fn roundtrip() -> Result<(), ParseError> {
         let cases: &[Key] = &[
-            Key::ChunkData("".to_string()),
-            Key::ChunkData("a".to_string()),
-            Key::ChunkRefs("".to_string()),
-            Key::ChunkRefs("a".to_string()),
+            Key::ChunkData(String::from("".to_string())),
```

Comment:
It's a bit annoying to me that there are so many ways to do this conversion! I've taken to doing `"".into()` just because it's the shortest.

Should we perhaps try to standardize on a preferred way to do this str-constant -> String conversion throughout, or not worry about it?

Reply:
Yeah I hate how many ways there are to convert to and fro in Rust -- or at least that's my perception so far, not knowing what they all do. Happy to standardize on into().